Mathlib/Data/Fin/Basic.lean

```diff
@@ -451,6 +458,8 @@ whose value is the original number. -/
 theorem val_cast_of_lt {n : ℕ} [NeZero n] {a : ℕ} (h : a < n) : (a : Fin n).val = a :=
   Nat.mod_eq_of_lt h
 
+theorem one_val_cast {n : ℕ} [NeZero n] (h : 1 < n) : (1 : Fin n).val = 1 := val_cast_of_lt h
```

Comment:
It does not work, I get the error `tactic 'rewrite' failed, did not find instance of the pattern in the target expression
  ↑↑1` because in the target expression I have `↑1`

Reply:
I made a suggestion which avoids it, but if you want to keep it I think it should be named `val_one_of_one_lt`